alexa_client/alexa_client/client.py

```diff
@@ -49,7 +49,10 @@ def synchronise_device_state(self):
                 device_state=self.device_manager.get_device_state(),
             )
 
-    def send_audio_file(self, audio_file, dialog_request_id=None):
+    def send_audio_file(
+        self, audio_file, dialog_request_id=None,
+        distance_profile=constants.CLOSE_TALK, audio_format=constants.PCM
```

Comment:
eventually a refactor to `device_manage` will be done that allows the device manager to handle the `distance_profile`, `audio_format` and device status